packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -31,6 +31,10 @@ module.exports = (themeOptions = {}) => {
       author: 'commercetools',
       productionHostname,
       betaLink: '.',
+      globalNotification: {
+        notificationType: false,
+        content: false,
```

Comment:
I don't think it's ideal to make the default for string properties "false", esp. if the data is exposed on GraphQL you get into ugly errors because gatsby fails to build the schema.  Either give it no default (not set or null) or one that has the right type